### ISSUE TITLE
Download speed estimations are likely too optimistic

### ISSUE THREAD

**Issue by Calinou** [pastelsky/bundlephobia#535]
**Please describe the feature/suggestion**

I saw that the download speed estimations were revised since I last visited Bundlephobia.

I'll take an example with [`react@17.0.2`](https://bundlephobia.com/package/react@17.0.2):

![image](https://user-images.githubusercontent.com/180032/123519871-1d59a580-d6ae-11eb-8e47-dccaa94c765f.png)

According to the tooltips, the download speed for "slow 3G" is 400 kilobytes/s and the download speed for "emerging 4G" is 7 megabytes/s. However, these download speeds seem to be overestimated compared to what you usually get in practice. Even with fixed connections such as ADSL2+, you can't quite reach a download speed of 7 MB/s. Yet ADSL2+ remains quite common in many parts of the world.

Back when 4G was not readily available in my area (I'm in France), it was rare for me to get download speeds above 50 KB/s in 3G.

I feel like kilo**bytes** and kilo**bits** may have been mixed up together here :slightly_smiling_face: 

**Describe the solution you'd like**

Change the download speed for "slow 3G" to 400 kilo**bits**/s (= 50 kilobytes/s) and the download speed for "emerging 4G" from to 7 mega**bits**/s (= 875 kilobytes/s). This reflects real world speeds more accurately, especially on smaller files since downloading the first bytes will often be slower than the rest of the file.

Having accurate estimations is important, as people will use Bundlephobia for decision making. It's important to show that many people still have slow connections today. Developers should not assume that everyone has good 4G or fiber available.

**Describe any alternatives you've considered**

N/A

**Comment by pastelsky** [comment]
I agree that the initial speeds I added may have been too optimistic (it was the KB / Kb issue as you mentioned). This has been fixed.

As for latency — I don't think bundlephobia is the right tool to indicate that since most people would have several hundred modules in their bundles, but only have them bundled into 3-4 files on average. The latency cost is paid per file, and if loading in parallel / pre-connecting, even that cost might not need to be paid.

However, I've added a small note about us not including latency in the tooltip. 

Thanks!
